spring-integration-redis/src/main/java/org/springframework/integration/redis/inbound/RedisInboundChannelAdapter.java

```diff
@@ -115,6 +123,9 @@ protected void onInit() {
 		}
 		adapter.afterPropertiesSet();
 		this.container.addMessageListener(adapter, topicList);
+		if (this.taskExecutor != null) {
+			this.container.setTaskExecutor(this.taskExecutor);
```

Comment:
Look, the `container` property is `final`, so we easily can inject `taskExecutor` from the setter directly to `container`.
I mean that I don't see reason in the extra property overhead.

Reply:
Fixed in 1215f1a.